### PR TITLE
Removed unused imports in layers modules

### PR DESCRIPTION
**Pull request by yardstick17**

### CODE REVIEW

File: tensorflow/python/layers/utils_test.py [hunk 1]
```diff
@@ -18,9 +18,10 @@
 from __future__ import division
 from __future__ import print_function
 
-from tensorflow.python.layers import utils
 from tensorflow.python.platform import test
 
+from tensorflow.python.layers import utils
```
Comment (yifeif) [inline tensorflow/python/layers/utils_test.py]: This one the original order was okay?

Comment (yardstick17) [reply tensorflow/python/layers/utils_test.py]: Yes. I have undone the import order change here.

Comment (yifeif) [reply tensorflow/python/layers/utils_test.py]: Thanks!

**Comment by yifeif** [comment]
@tensorflow-jenkins test this please

**Comment by yifeif** [comment]
py3 rerun at http://ci.tensorflow.org/job/tensorflow-pull-requests-cpu-python3/6319/

**Comment by yifeif** [comment]
gpu rerun at https://ci.tensorflow.org/job/tensorflow-pull-requests-gpu/6681/

**Review by yifeif — APPROVED** [review]

**Pull request merged**